README.md

```diff
@@ -49,11 +49,15 @@ Push your local development database backup up to staging:
 
     staging restore development
 
-Deploy from master, and migrate and restart the dynos if necessary:
+Deploy from master to production
+(or any branch to staging or a feature environment),
```

Comment:
I think it might be clearer to separate these out, for example

```md
Deploy from master to production
and migrate and restart the dynos if necessary:

    production deploy

Deploy the current branch to staging
or other feature environment
and migrate and restart the dynos if necessary:

    staging deploy

_Note that deploys to non-production environments use `git push --force`._
```